cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaAgentClient.java

```diff
@@ -98,6 +98,33 @@ private HttpClient createHttpClient() {
         }
     }
 
+    private KeyStore getTrustStore() throws CertificateException, KeyStoreException, IOException, NoSuchAlgorithmException {
+        final CertificateFactory caCertFactory = CertificateFactory.getInstance("X.509");
```

Comment:
Don't we have some defined Constants for `X.509` or `JKS`? :) If not should we introduce them to eliminate redundancy in the project?